Makefile

```diff
@@ -306,22 +306,26 @@ serve:
 	 @echo "INFO     Starting a server at http://localhost:8000/"
 	 @cd build/website && python3 -m http.server 8000
 
+.PHONY: upload
+upload-mirror:
```

Comment:
What is upload-mirror supposed to do?

Reply:
We currently upload to aclweb.org (main) and aclanthology.org (mirror). Maybe I'm misunderstanding your question.

Once we've transitioned to the latter, we'll handle everything on aclweb.org with 301 redirects.

Reply:
I just meant that the purpose was not clear before you added a comment.